### PR TITLE
Fix minor bugs

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ function downloadAgent() {
     payload.owner = $inputs[1].value;
```
Comment (DilanUA) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:146]: Use double quoted object key access for reading JSON attributes as it's considered to be more reliable.
Ex: person.name as person["name"]

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ function downloadAgent() {
...
 
     var connectedCupRegisterURL = "/connectedcup_mgt/connectedcup/cup/register?" +
-                                                        "name=" + payload.name + "&owner=" + payload.owner;
+                                                        "name=" + encodeURI(payload.name) + "&owner=" + payload.owner;
```
Comment (DilanUA) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:149]: fix formatting issues on indentation.

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js (hunk 1 above)
Comment (charithag) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:146]: How bracket notation could be more reliable? It is all about the language semantics. Using dot notation is permitted in JS as long as attribute name is a valid identifier.
http://stackoverflow.com/a/4968448/1391687

File: modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js (hunk 2 above)
Comment (charithag) [inline modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js:149]: Fixed in PR #179

**Pull request merged**